lib/cdo/pardot.rb

```diff
@@ -64,6 +66,30 @@ class Pardot
   class InvalidApiKeyException < Exception
   end
 
+  # Deletes a list of prospects from Pardot. For Pardot API documentation, see
+  # http://developer.pardot.com/kb/api-version-4/prospects/#using-prospects.
+  # @param [Array[Integer]] prospect_ids of the prospects to be deleted.
+  # @return [Array[Integer]] the set of prospect_ids that failed to be deleted.
+  def self.delete_pardot_prospects(prospect_ids)
```

Comment:
Pardot has been very UNresponsive so far. I wouldn't spend time pushing on them on this unless we really needed it. The only concerns I'd have is API round trip time is about 4 seconds and we have a 25K/day call limit. So if we were deleting thousands of users in a batch frequently we might care. Otherwise deletion one at a time is fine.